src/main/java/ca/uhn/fhir/jpa/starter/interceptor/LoggerInterceptor.java

```diff
@@ -15,32 +12,56 @@
 import ca.uhn.fhir.rest.api.server.RequestDetails;
 import ca.uhn.fhir.rest.api.server.ResponseDetails;
 import ca.uhn.fhir.rest.server.servlet.ServletRequestDetails;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
 
 @Interceptor
-public class CorrelationIdInterceptor {
+public class LoggerInterceptor {
   private static final String X_CORRELATION_ID = "X-Correlation-Id";
+  private static final String CORRELATION_ID = "correlationId";
+  private static final String X_LAUNCH_ID = "X-Launch-Id";
+  private static final String X_APP_NAME = "X-App-Name";
+  private static final String LAUNCH_ID = "launchId";
+  private static final String APP = "app";
+
 
   @Hook(Pointcut.SERVER_INCOMING_REQUEST_PRE_PROCESSED)
   public boolean incomingRequestPreProcessed(HttpServletRequest request,
       HttpServletResponse response) {
-    String corelationId = ObjectUtils.isNotEmpty(request.getHeader(X_CORRELATION_ID))
+	 String correlationId = ObjectUtils.isNotEmpty(request.getHeader(X_CORRELATION_ID))
         ? request.getHeader(X_CORRELATION_ID)
         : UUID.randomUUID().toString();
-    MDC.put(X_CORRELATION_ID, corelationId);
-    return true;
+	MDC.put(CORRELATION_ID, correlationId);
```

Comment:
Here is another tab that should be spaces. ex: formatting

There are lots of these. The existing code in this file had spaces. Your new/updated code should match.